### PR TITLE
Bump StencilSwiftKit version to version 1.0.2

### PR DESCRIPTION
**Pull request by diogot**
* [X] I've added an entry in the `CHANGELOG.md` file to explain my changes and credit myself
  * [X] Add the entry in the appropriate section (Bug Fix / Nw Feature / Breaking Change / Internal Change)
  * [X] Add a period and 2 spaces at the end of your short entry description
  * [x] Add links to your GitHub profile to credit yourself and to the related PR and issue after your description

---

My contribution to #290

- Bump StencilSwiftKit to version 1.0.2
- User Xcode 8.3 in CircleCI

### CODE REVIEW

File: Pods/Local Podspecs/StencilSwiftKit.podspec.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "StencilSwiftKit",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (djbe) [inline Pods/Local Podspecs/StencilSwiftKit.podspec.json]: That can't be right?

Comment (diogot) [inline Pods/Local Podspecs/StencilSwiftKit.podspec.json]: 🤔, indeed.
Let me check.

Comment (diogot) [inline Pods/Local Podspecs/StencilSwiftKit.podspec.json]: All podspecs in `Pods/Local Podspecs` belong to pods in the file system.
Given that currently there are no local pods in `Podfile` those probably are leftovers from the development process.
I will remove them from git.

**Comment by diogot** [comment]
Thanks!

File: rakelib/utils.rake [hunk 2]
```diff
@@ -68,9 +68,11 @@ class Utils
 
   # select the xcode version we want/support
   def self.version_select
-    xcodes = `mdfind "kMDItemCFBundleIdentifier = 'com.apple.dt.Xcode' && kMDItemVersion = '8.*'"`.chomp.split("\n")
+    version = '8.*'
+    Rake.sh 'mdimport /Applications/Xcode* && sleep 1'
```
Comment (diogot) [inline rakelib/utils.rake]: Sorry, I forgot to remove here.
It's fine in [StencilSwiftKit](https://github.com/SwiftGen/StencilSwiftKit/blob/master/rakelib/utils.rake#L72).

**Comment by AliSoftware** [comment]
Thanks a lot for contributing to SwiftGen & StencilSwiftKit! 👍 
To thank you for your work and for taking time to improve the tool, I've invited you to join SwiftGen's GitHub organization's "CoreContributors" team 😃 

No pressure to accept nor any obligation in that! It will just give you the ability to help triage issues or push directly branches to the repo (instead of your fork)… in case you want to contribute again in the future 😉

For more information, read [COMMUNITY.md](COMMUNITY..md) and [CONTRIBUTING.md](CONTRIBUTING.md) and if you have any questions don't hesitate to reach out!

**Comment by diogot** [comment]
Awesome, thanks!

**Pull request merged**